ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_norm.py

```diff
@@ -43,3 +43,38 @@ def test_paddle_layer_norm(
         bias=offset[0],
         epsilon=eps,
     )
+
+
+# bach_norm
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.bach_norm",
+    values_tuple=_generate_data_layer_norm(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
```suggestion
        available_dtypes=helpers.get_dtypes("valid"),
```

`valid` automatically returns the datatypes that are supported by the function - if it's only `float` then that's what `valid` will return.

Reply:
Yup.